website/common/locales/en/tasks.json

```diff
@@ -16,9 +12,6 @@
   "theseAreYourTasks": "These are your <%= taskType %>",
   "habit": "Habit",
   "habits": "Habits",
-  "newHabit": "New Habit",
-  "newHabitBulk": "New Habits (one per line)",
-  "habitsDesc": "Habits don't have a rigid schedule. You can check them off multiple times per day.",
```

Comment:
this one, alongside `todosDesc`, `dailysDesc` and `rewardsDesc` is still used here https://github.com/HabitRPG/habitica/blob/develop/website/client/src/components/tasks/column.vue#L77

I think an "easy" way to check for these types of usages is:

- on the client translated strings are always generated with a call to `$t(stringName)`, so if you search for `$t(` followed by a character that is not `'` it means that the string key is being dynamically created (either by passing a variable with the string key or by constructing a string using ` \` `
- for the server / common code it's the same but using `.t(` instead of `$t(`

Reply:
Let me know if you think this can be a viable option to solve the issue, also just noticed there are a bunch of cases (the only ones in the entire codebase) of `+` being used to construct i18n keys in this file https://github.com/HabitRPG/habitica/blob/develop/website/client/src/components/inventory/items/cards-modal.vue